packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -430,7 +437,8 @@ class DatePicker extends React.Component {
         >
           <ReactDatePicker
             {...customProps}
-            withPortal={this.state.showPortalPicker}
+            withPortal={(inline) ? false : this.state.showPortalPicker}
```

Comment:
Should the showPortalState be default to inline value?

```
  constructor(props) {
    super(props);
    const activeBreakpointOnMount = activeBreakpointForSize(window.innerWidth);
    this.state = {
      selectedDate: DateUtil.defaultValue(props),
      showPortalPicker: !inline || activeBreakpointOnMount === 'tiny' || activeBreakpointOnMount === 'small',
      prevPropsSelectedDate: props.value || props.selectedDate,
    };

```

Also do we need the logic in `this.handleBreakpointChange` to prevent a state update when inline is true?

Reply:
updated : [63be00c147](https://github.com/cerner/terra-framework/pull/1134/commits/8ec0355f75cba2e79d27fbd326b343bec97afba4#diff-06f07d3cca7012107347dc63be00c147)